forte/base_classes/active_space_method.cc

```diff
@@ -356,6 +358,71 @@ std::vector<std::vector<double>> ActiveSpaceMethod::compute_transition_dipole_sa
     }
     psi::outfile->Printf("\n    %s", dash.c_str());
 
+    // Doing SVD for transition reduced density matrix
+
+    // Several necessary matrix and constants for LAPACK SVD
```

Comment:
Why not using libmint's svd function here?

Reply:
I changed to libmint svd.